extract/common.py

```diff
@@ -1,6 +1,7 @@
 import re
 import tempfile
 import textract
+from bs4 import BeautifulSoup as bs
```

Comment:
do we still need bs4 for the extractors?

Reply:
No, I just missed this line when I was cleaning up. Next push I'll have it removed.